app/elements/coder/code-editor/file-explorer/file-explorer.html

```diff
@@ -44,15 +48,16 @@
 				'file-selected': '_fileSelected'
 			},
 			ready: function(){
+				this.rootItem = {name: '/', value: {}}
```

Comment:
In the UI only file names are shown, therefore a `/` should be nowhere to be seen.